### PR TITLE
Add gallery.css

### PR DESCRIPTION
**Pull request by oliviale**
### New Stylesheet Submission:

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other files using the same name as yours?
- [x] Have you filled in at least the required data for `title`, `author`, and `stylesheet`?
- [x] Is the value of `stylesheet` a full URL that is publicly accessible and renders an unminified, compiled CSS file?
- [x] Do your styles pass proper contrast for accessibility?
- [x] Have you included `@media (prefers-reduced-motion: reduce)` to remove animations on `*,*::before,*::after`?
- [x] Did you style the `.skip-link`?
- [x] Have you checked on a real mobile device to ensure your responsive design doesn't cause overlaps or overflow scrolling?

### CODE REVIEW

File: src/_data/styles/gallery.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Gallery",
+  "author": "Olivia Ng",
+  "stylesheet": "https://assets.codepen.io/567707/gallery.css",
+  "twitter": "meowlivia_",
```
Comment (5t3ph) [inline src/_data/styles/gallery.json:5]:
```suggestion
  "twitter": "meowlivia_"
```

**Comment by 5t3ph** [comment]
Hi @oliviale - unfortunately your current stylesheet link will not work with the Style Stage build (and I don't want to delay adding this in order to make a change). If you use CodePen, it needs to be a public pen so that you URL can be `[pen URL].css` (which may 404 on direct access but it will work during the build process). If you'd like to privately host the URL instead of a public pen, you can do a GitHub gist and provide the "raw" code URL, or you can host it off of a domain you own. Looking forward to your update so I can review it!

**Comment by oliviale** [comment]
Hi @5t3ph! Gotcha, fixed using a gist URL instead :)

**Comment by 5t3ph** [comment]
@oliviale 😍 Oh my, this is so clever! There is one contrast issue which is the `h1` - you can either darken the interior color so that it's at least 3:1 with _both_ of the grey backgrounds it's over (I realize it's one background technically but it appears like two), or adjust your `box-shadow` so it essentially allows at least a 1px inset that meets 3:1 contrast all the way around the letterform over both colors. 

Head's up - I think the gist link will change with the update so you may have to recommit an edited URL.

Super excited, I think if you are able to get this in today it will be style number 50, which will kick off a giveaway ;)

**Comment by olivia-ng** [comment]
@5t3ph Ahh ok, so I fixed it in the gist, and I used the non-versioned URL, so it will always take the latest version so I don't need to update the URL.

**Review by 5t3ph — APPROVED** [review]
@oliviale awesome, so glad you had a minute to do that right away, publishing now!

Thank you for participating! If you haven't already, consider subscribing for updates to learn about new styles and features _and_ to get the giveaway details once this publishes: https://stylestage.dev/subscribe/

**Comment by olivia-ng** [comment]
Awesomeee, thanks for all your hard work 😄  And subscribed!

**Pull request merged**